discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -95,3 +106,18 @@ impl Ops for BreakingNews {
         Ok(res)
     }
 }
+
+fn spawn_headlines_request(
+    client: Arc<Client>,
+    market: Market,
```

Comment:
should we wrap each `market` in an `Arc`? otherwise this might become costly to clone since we have to do this for each query.

Reply:
We don't do query that often and market is 2 strings, with two char inside. the alloc will cost, but this is nowhere near a hot path.